### ISSUE TITLE
Assertion Failed: strCommonRedistForeignInstallPath != ""

### ISSUE THREAD

**Issue by ChefNiko** [ValveSoftware/Proton#4830]
Game: The Elder Scrolls V: Skyrim Special Edition
Proton: 6.3-4 and experimental

Observed: The game fails to launch
Expected: The game launches

OS: Kubuntu 21.04 (5.11.0-17-generic x64)

Additional Info:
Looking at the terminal output, the failure seems to be that the path for "drive_c" is set incorrectly. I tried to update it using winecfg, but that also will not load with the path being broken. Even with that assert failing, the path shouldn't be set so brokenly. Perhaps falling back to a known working path would be a good idea, because at least that would be easier to "fix".

The problem seems to be based on the failed assert mentioned in the title. I found some similar issues, but I'm not sure the fix is exactly the same. E.g. https://github.com/ValveSoftware/Proton/pull/4409

Full terminal output log attached.
[steam.log](https://github.com/ValveSoftware/Proton/files/6493778/steam.log)

**Comment by kisak-valve** [comment]
Hello @ChefNiko, what filesystem are you using with `/media/niko/Storage-R1/Games/SteamLibrary`? I suspect it is NTFS formatted, which is known to have finicky interactions with Proton.

**Comment by ChefNiko** [comment]
Yes, it is NTFs filesystem. I'm sorry to hear Proton doesn't interact with it well, that's disappointing.

Just from poking around a bit, I see that the path it is looking for does not exist, but for another game, it is a shortcut to another folder.

This leads me to believe that the install / initial config failed without notification in the GUI.

**Comment by kisak-valve** [comment]
The general recommendation is to use a native Linux filesystem with Proton, but you could try giving https://github.com/ValveSoftware/Proton/wiki/Using-a-NTFS-disk-with-Linux-and-Windows a read or [searching for other NTFS-related issues](https://github.com/ValveSoftware/Proton/search?q=ntfs&type=issues) on this issue tracker for a workaround.

**Comment by ChefNiko** [comment]
Based on your recommendation, I moved Proton to a native Linux file system. However, it would be great if Proton didn't install itself on to a non-recommended file system by default, or at least warned me about it.

Unfortunately, even after a reinstall of the game as well, it failed to launch with the following error printed on the terminal:
'wine: could not open working directory L"unix\\home\\niko\\.steam\\debian-installation\\", starting in the Windows directory.'

Assuming 'L"unix"' is some sort of translation, it may not be working correctly, as /home/niko/.steam/debian-installation/ is a valid directory. It may also be some other error, as the message doesn't really say what the problem is.

I tried to uninstall proton so I could re-install it directly on to a linux file system, which didn't work. Unfortunately, the message in the UI was unhelpful, but the terminal output had a usable message.

After re-installing proton on to a linux file system, I attempted to launch the game again. The dialog box said "Ready to Play" but the "Play Game" button was disabled.

After a while, the same errors appeared in the terminal:
'wine: could not open working directory L"unix\\home\\niko\\.steam\\debian-installation\\", starting in the Windows directory.'

I tried another game which used to work, and it printed the same errors about not being able to open the working directory.... but did display some other messages as well. After a while of just seeing the same directory error, I terminated the process.

At this point, I ran the game through wine, using my default prefix, and it launched fine (besides how difficult it is to run Steam through wine), so this seems to be a steamplay / proton specific problem. 

Based on this testing, it seems like steam play is unusable at this point.
